components/Modal/Modal.jsx

```diff
@@ -0,0 +1,86 @@
+import {useState} from 'react';
+import PropTypes from 'prop-types';
+import {View, Text, FlatList, TouchableOpacity} from 'react-native';
+import Modal from 'react-native-modal';
+import {theme} from '../../assets/theme/theme';
+import StarIcon from '../../assets/icons/Star.svg';
+import styles from './style';
+
+const CustomModal = ({isModalVisible, setModalVisible, navigation}) => {
+  const [activeStarsColor, setActiveStarsColor] = useState(false);
+  const [numberOfStarsPlaced, setStarsPlaced] = useState(0);
+
+  const changeColorOfStars = count => {
+    setActiveStarsColor(true);
+    setStarsPlaced(count);
+  };
+
+  return (
+    <Modal
+      isVisible={isModalVisible}
+      coverScreen={true}
+      transparent={true}
```

Comment:
coverScreen
      transparent
      
     instead of 
     
      coverScreen={true}
      transparent={true}

Reply:
I corrected it.